src/Microsoft.Health.Fhir.Core/Configs/ExportJobConfiguration.cs

```diff
@@ -31,8 +31,16 @@ public class ExportJobConfiguration
             Justification = "Set from an environment variable.")]
         public string StorageAccountUri { get; set; } = string.Empty;
 
+        /// <summary>
+        /// Sets the maximum number of running export jobs allowed across all instances.
+        /// </summary>
         public ushort MaximumNumberOfConcurrentJobsAllowed { get; set; } = 1;
```

Comment:
> = 1; [](http://example.com/codeflow?start=72&length=5)

I am not sure about the value this setting provides when per instance setting is in place. True parallelism should be driven from number of worker processes (equal to number of VMs). In any case I would set this value pretty high.